ncm-dirperm/src/main/perl/dirperm.pm

```diff
@@ -31,6 +35,24 @@ sub Configure($$@) {
     # Define paths for convenience.
     my $base = "/software/components/dirperm";
 
+    my $filesystems = "/system/filesystems";
```

Comment:
fs->mounted returns if the fs is mounted. 
There is nothing in ncm-lib-blockdevices for what should be mounted. this is done in ncm-fstab.
Checking if path is below it is not there afaik